pkg/importer/importer.go

```diff
@@ -1132,6 +1136,7 @@ type importerAcct struct {
 	lastRunErr   error
 	lastRunStart time.Time
 	lastRunDone  time.Time
+	usePaidAPI   bool // whether to use the paid API(s) when retrieving data
```

Comment:
whether to use API endpoints (if any) that may require paying/credits

Reply:
Now there's a separate bool (on the higher level, of the importer) that specifies whether or not there are paid APIs. This boolean now controls whether or not to use them.